browser/lib/newNote.js

```diff
@@ -28,9 +29,9 @@ export function createMarkdownNote (storage, folder, dispatch, location, params,
         note: note
       })
 
-      hashHistory.push({
+      history.push({
```

Comment:
Nit: Should I change this to `dispatch(push(..))` so it's consistent to the other locations? Then we could also remove the import of history.

Reply:
I think so. It should be better.